.github/workflows/GHA-Functional-Tests.yaml

```diff
@@ -89,7 +89,7 @@ jobs:
           sed -i -e "s|jdk8=8|jdk8=${ORG_GRADLE_PROJECT_jdk8}|
           s|jdk11=11|jdk11=${ORG_GRADLE_PROJECT_jdk11}|
           s|jdk17=17|jdk17=${ORG_GRADLE_PROJECT_jdk17}|
-          s|jdk19=19|jdk19=${ORG_GRADLE_PROJECT_jdk19}|" gradle.properties.gha
+          s|jdk20=20|jdk20=${ORG_GRADLE_PROJECT_jdk20}|" gradle.properties.gha
```

Comment:
There are a few places that we set a envvar GRADLE_PROPERTIES, and the ORG_GRADLE_PROJECT_jdk19 should be changed there also.